src/App.tsx

```diff
@@ -25,15 +33,19 @@ function App() {
     setSelectedFile(file);
   }, []);
 
+  const handleCenterPanelResize = () => {
+    pdfViewerRef.current?.updateWidth();
```

Comment:
I feel that we can inline this

Reply:
I usually prefer the bracket syntax for void functions because the inline version would be equivalent to
```typescript
const handleCenterPanelResize = () => {
    return pdfViewerRef.current?.updateWidth();
}
```
which is semantically different. Here it wouldn't change anything because `updateWidth` is a void method so it returns undefined. What are you thoughts on that? Should we inline it anyway?